crypto/bls/private.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package bls
+
+import (
+	"errors"
+
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
+)
+
+const PrivateKeyLen = bls.SecretKeyLen
+
+var (
```

Comment:
nit: you should be able to do `var errFailedPrivateKeyDeserialize = errors.New("couldn't deserialize secret key")`